client/cloud_credentials_test.go

```diff
@@ -80,7 +80,8 @@ var _ = Describe("CloudCredentials", func() {
 			mockOrganizationIdCall(organizationId)
 
 			payloadValue := AwsCredentialsValuePayload{
-				RoleArn: "role",
+				RoleArn:  "role",
+				Duration: 1,
```

Comment:
It's not one of the allowed values, shouldn't the test fail on this?

Reply:
This is enforced at the schema/resource level. Not the API level.